apps/condo/domains/common/components/ActionBar.tsx

```diff
@@ -24,7 +24,7 @@ const ActionBar: React.FC<IActionBarProps> = ({ children, fullscreen = false })
     return (
         <Affix offsetBottom={24} style={{ width: fullscreen ? '100%' : 'unset' }}>
             <div css={actionBar}>
-                <Space wrap={true} size={[40, 24]}>
+                <Space wrap={true} size={[40, 24]} style={{ marginBottom: '0 !important' }}>
```

Comment:
Bad practise. We shouldnt use !important here. Use common ant'd components and props.